media/themes/Modern/modern.msstyles/lang/en-GB.rc

```diff
@@ -14,14 +14,14 @@ STRINGTABLE
 
 STRINGTABLE
 {
-3000, "Regular Fonts"
+3000, "Normal"
```

Comment:
may I ask why also editing these stuff🤣

Reply:
Nothing other than consistency. 

This theme is the only one that uses "Regular Fonts" instead of "Normal"